gma/src/common/native_ad_internal.cc

```diff
@@ -59,6 +61,20 @@ Future<AdResult> NativeAdInternal::GetLoadAdLastResult() {
       future_data_.future_impl.LastResult(kNativeAdFnLoadAd));
 }
 
+void NativeAdInternal::insert_image(const NativeAdImage& image,
+                                    std::string image_type) {
```

Comment:
const std::string&

Reply:
Changed it to a bool as we only have two image types.